docs/src/main/paradox/client-side/request-level.md

```diff
@@ -111,3 +111,14 @@ URIs or a valid `Host` header. The host connection pool will automatically add a
 
 For a super-pool flow this is not the case. All requests to a super-pool must either have an absolute URI or a valid
 `Host` header, because otherwise it'd be impossible to find out which target endpoint to direct the request to.
+
+
+## Collecting headers from a server response
+
+Sometimes we would like to get only headers of specific type which are sent from a server. In order to collect headers in a type safe way  akka-http API provides a type for each HTTP header. Here is an example for getting all cookies set by a server (Set-Cookie header):
```

Comment:
- There is a double space in `way  akka-http`.
- iirc, the common way is Akka HTTP when referred to the project.
- Wrap `Set-Cookie` in back-ticks so it's clear that this is the name of the cookie.